tidb-cloud/tidb-node-group-management.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Manage TiDB Node Groups
+summary: Learn about how to manage TiDB node groups and their endpoints to isolate your business workload.
+---
+
+# Manage TiDB Node Groups
+
+This document describes how to manage TiDB node groups and their endpoints to isolate your business workload using the [TiDB Cloud console](https://tidbcloud.com/).
+
+> **Note**:
+>
+> The TiDB Node Group feature is **NOT** available for TiDB Cloud Serverless clusters.
+
+Currently, the TiDB Node Group feature is only available upon request. To request this feature, perform the following steps:
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
+2. Click **?** in the lower-right corner.
+3. Click **Request Support**. 
+4. In the **Subject** field, fill in **Apply for TiDB Node Group feature**.
+5. Click **Submit**.
+
+## Terms
+
+- TiDB Node Group: A TiDB node group manages the grouping of TiDB nodes and maintains the mapping between endpoints and TiDB nodes.
+
+    - Each TiDB node group has a unique endpoint. 
+    - When you delete a TiDB node group, the related network setting (such as private link and IP access list) will be deleted too. 
+
+- Default Group: When a cluster is created, a default TiDB node group is created. Therefore, each cluster has a default group. The default group cannot be deleted. 
+
+## Prerequisites
+
+- You have a [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) cluster deployed on AWS.
+- You are in the **Organization Owner** or **Project Owner** role of your organization. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).
+
+## Create a TiDB node group
+
+To create a TiDB node group, perform the following steps:
+
+1. In the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, and then click the name of your target cluster to go to its overview page.
+2. Click **Nodes** in the left navigation pane.
+3. Click **Modify** in the upper-right corner. The **Modify Cluster** page is displayed.
+4. On the **Modify Cluster** page, click **+** to add a new TiDB node group as follows. You can also use the default group directly.
+
+    - TiDB
+        - **vCPU + RAM**: choose the [TiDB size](/tidb-cloud/size-your-cluster.md#size-tidb) you need. Only TiDB nodes with 8 vCPU and 16 GiB memory or higher specifications are supported.
+        - **Node Groups**: click **+** to create a new TiDB node group. You can also use the default group and enter the number of TiDB nodes in the **Default Groups** field.
```

Comment:
```suggestion
        - **Node Groups**: click **+** to create a new TiDB node group. You can also use the default group and enter the number of TiDB nodes in the **DefaultGroup** field.
```
The field name in the image `media/tidb-cloud/tidb-node-group-change-node-count.png` is "DefaultGroup"
<img width="278" alt="image" src="https://github.com/user-attachments/assets/4976cdb5-c014-47a8-8086-90842313d18e" />